Gemfile

```diff
@@ -1,8 +1,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
```

Comment:
Hi @derrek,
I definitely can, but I moved these files from the `Gemfile` into `street-address.gemspec`, which is the preferred method (as much as their is a preferred method; I should restart that having gem dependencies, including dev dependencies, in the gemspec file is the way bundler sets up new gems...)

with that said, if you want me to go back to having dev dependencies in the gemfile rather than the gemspec file, I can do that.

Ref: http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/